playwright/Dockerfile.debian

```diff
@@ -0,0 +1,71 @@
+FROM registry.lil.tools/library/debian:buster
+ENV LANG=C.UTF-8 \
+    LC_ALL=C.UTF-8 \
+    PYTHONUNBUFFERED=1 \
+    PIP_NO_CACHE_DIR=off \
+    PIP_DISABLE_PIP_VERSION_CHECK=on \
+    PIP_SRC=/usr/local/src \
+    PIPENV_HIDE_EMOJIS=true \
+    PIPENV_NOSPIN=true \
+    OPENSSL_CONF=/etc/ssl
+
+# Get build dependencies and packages required by the app
+RUN apt-get update \
+    && apt-get install -y wget \
+    && apt-get install -y curl
+
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - \
+    && apt-get install -y nodejs
+
+RUN apt-get install -y git \
+    && apt-get install -y --no-install-recommends openssh-client \
+    && npm install -g yarn \
+    && apt-get install -y python3-pip \
+    && apt-get install -y python3-dev \
+    && apt-get install -y virtualenv \
+    && apt-get install -y gnupg \
+    && apt-get install -y xvfb \
+    && apt-get install -y libjpeg62-turbo-dev \
+    && apt-get install -y libfontconfig1 \
+    && apt-get install -y libglib2.0-0 \
+          libnss3 \
+          libnspr4 \
+          libatk1.0-0 \
+          libatk-bridge2.0-0 \
+          libcups2 \
+          libdbus-1-3 \
+          libxkbcommon0 \
+          libxcomposite1 \
+          libxdamage1 \
+          libxrandr2 \
+          libgbm1 \
+          libgtk-3-0 \
+          libpango-1.0-0 \
+          libcairo2 \
+          libgdk-pixbuf2.0-0 \
+          libasound2 \
+          libatspi2.0-0
+
+# Install commonly used web fonts for better screen shots.
+RUN echo "deb http://httpredir.debian.org/debian buster main contrib" > /etc/apt/sources.list \
+    && echo "deb http://security.debian.org/ buster/updates main contrib" >> /etc/apt/sources.list \
+    && echo "ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true" | debconf-set-selections \
+    && apt-get update \
+    && apt-get install -y ttf-mscorefonts-installer \
+    && apt-get install -y fonts-roboto
+
+
+# Setup pseudo-home
+RUN mkdir -p /playwright
+WORKDIR /playwright
+
+COPY playwright/Pipfile /playwright/Pipfile
+COPY playwright/Pipfile.lock /playwright/Pipfile.lock
+RUN pip3 install -U pip \
+    && pip install pipenv \
+    && pipenv --python 3.7 install --ignore-pipfile
+
+RUN pipenv run python -m playwright install
+
+# To support firefox and webkit
+# RUN pipenv run python -m playwright install-deps
```

Comment:
This currently fails and so it's commented out. I may be able to do a bit more work to get FF and webkit to work without this line though.